orc/graphs/mcp_executor.py

```diff
@@ -74,6 +75,61 @@ def _emit_progress(self, tool_name: str, percent: int) -> None:
             return
         self.progress_streamer.emit_progress(step, STEP_MESSAGES[step], percent)
 
+    _TS = re.compile(r'_(\d{13}|\d{10})(?=\.)')
```

Comment:
@LuisMRL1997 I’m not sure if this file is still being called after the latest PRs from @namtran6701 were integrated.

The mcp_executor is initialized in orc/graphs/main_2.py, and that module is only imported here:

from orc.graphs.main_2 import create_conversation_graph


inside orc/new_orchestrator.py, which no longer seems to be used.

You might want to double-check with @namtran6701 whether this file is still referenced anywhere or if it can be safely removed.